### PR TITLE
Link SoftBody tutorial in docs

### PR DESCRIPTION
**Pull request by Jummit**
- [x] Document some more properties/functions and also make `ray_pickable` a property (set_ray_pickable and is_ray_pickable are already there).

### CODE REVIEW

File: doc/classes/SoftBody.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 		A deformable physics body. Used to create elastic or deformable objects such as cloth, rubber, or other flexible materials.
 	</description>
 	<tutorials>
+		<link>http://docs.godotengine.org/en/latest/tutorials/physics/soft_body.html</link>
```
Comment (akien-mga) [inline doc/classes/SoftBody.xml]: Should be `https`.

File: scene/3d/soft_body.cpp [hunk 2]
```diff
@@ -700,6 +702,8 @@ SoftBody::SoftBody() :
 		simulation_started(false),
 		pinned_points_cache_dirty(true) {
 
+	ray_pickable = true;
```
Comment (akien-mga) [inline scene/3d/soft_body.cpp]: This should be initialized like other fields in the list.

Comment (Jummit) [reply scene/3d/soft_body.cpp]: I "copied" it from here:
https://github.com/godotengine/godot/blob/99cee9038bf4fe049991d0048de8b8d8955bd9c7/scene/3d/collision_object.cpp#L342-L350

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**